server/controllers/user.controller.js

```diff
@@ -1,30 +1,41 @@
-const express = require('express');
-const router = express.Router();
-const userModel = require("../models/user.model");
+'use strict';
+const responseUtil = require('../utils/responseUtils');
+const MessageUtil = require('../utils/messageUtil');
+const userService = require("../services/user.service");
 const md5 = require('md5');
```

Comment:
We could use inbuilt hashing available on mongo db to encrypt password to db. But not a immediate change.